### DISCUSSION TITLE
Dynamic slides addition

### DISCUSSION THREAD

**Discussion by DawidJaneczkoMB** [davidjerleke/embla-carousel#484] [Help] [answered]
Hey! I've been trying to do dynamic addition of slides to the carousel. I want to be able to add a slide while swiping, however the carousel doesn't seem to react to slides number change. I also tried .reInit() and then it 'sees' new slides, but there is no scrolling animation. Is it possible with Embla to achieve the dynamic addition of slides?
https://codesandbox.io/s/brave-proskuriakova-o0pnyr?file=/src/Carousel.js

**Comment by davidjerleke** [comment, marked as answer]
Hi @DawidJaneczkoMB,

Do you mean that you want to add slides while the carousel is scrolling? That adding slides shouldn't stop the carousel? Have you missed the [infinite scroll example](https://www.embla-carousel.com/examples/static/#infinite-scroll) on the examples page?

Best,
David

**Reply by DawidJaneczkoMB** [reply]
Ah yes, I must have missed it. Thank you!
